lib/updatehub/updatehub_firmware.c

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2018 O.S.Systems
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "updatehub_firmware.h"
+
+bool updatehub_get_firmware_version(char *version)
```

Comment:
Better would be to put the version buffer length as a second parameter

Reply:
We can do that, however, we are using the `BOOT_IMG_VER_STRLEN_MAX` constant which is defined elsewhere.

Reply:
Sure and that works for now, it is just that if someone calls this function but uses different length for the version, then invalid memory access might occur. If the length is part of the function call, that risk of that wold be minimized.